lib/processes/applyGains.cpp

```diff
@@ -25,14 +25,16 @@ REGISTER_KOTEKAN_PROCESS(applyGains);
 applyGains::applyGains(Config& config, const string& unique_name,
                        bufferContainer& buffer_container) :
     KotekanProcess(config, unique_name, buffer_container,
-                   std::bind(&applyGains::main_thread, this)) {
+                   std::bind(&applyGains::main_thread, this)),
+    in_buf(get_buffer("in_buf")),
+    out_buf(get_buffer("out_buf")),
+    frame_id_in(frameID(in_buf)),
```

Comment:
Does this now work if you just use `frame_id_in(in_buf)` without the intermediate object being constructed?

Reply:
Thanks! Fixed.